### DISCUSSION TITLE
Remove librechat* from .dockerignore

### DISCUSSION THREAD

**Discussion by ai-made-approachable** [danny-avila/LibreChat#1759] [Troubleshooting]
### What happened?

The fact that librechat* is in the .dockerignore file completely invalidates the whole tutorial of setting up custom endpoints:
https://docs.librechat.ai/install/configuration/custom_config.html

So please remove it ...

### Steps to Reproduce

Follow these instructions to setup a custom endpoint: https://docs.librechat.ai/install/configuration/custom_config.html
And find out that even with the docker compose overrite file etc. your librechat.yaml will never be found.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
>  And find out that even with the docker compose overrite file etc. your librechat.yaml will never be found.

This is not the case for myself along with many other people with container setups using docker desktop, portainer, k8s, etc.

Can you walk me through your setup and maybe I can help?

**Comment by ai-made-approachable** [comment]
You can find all the details in this thread: https://discord.com/channels/1086345563026489514/1203320565679530024
Running the application in railway. Removing "librechat*" from the dockerignore made it finally work for me.

**Reply by danny-avila** [reply]
I see, that is an issue specific to railway, maybe railway setup can work on a separate branch with a different dockerignore file @Berry-13
